README.md

```diff
@@ -38,6 +38,30 @@ NOTE: You will still need to configured your name/email globally the first time
 git config --global user.name "Homer Simpson"
 git config --global user.email homer@springfieldpower.com
 ```
+
+## Sample Molecule Testing Role
+
+A sample role has been provided in the roles/backup_file directory to experiment with Test Driven Development using Molecule and OpenShift DevSpaces. A molecule verifier has been configured to test that the role functions, but the role's main.yml has not been written yet to pass the tests. A working example of the tasks/main.yml is available at tasks/answers.yml.
+
+### Automation requirements
+1. Make a backup of a file identified using the backup_file_source variable
+2. The backup should be stored in the directory identified by the backup_file_dest_folder variable
+3. If the backup directory doesn't exist, it should be created and writable
+4. The backup file should have a suffix appended such as '.bak' which is identified by the backup_file_dest_suffix variable
+
+### To begin development
+1. Click the three horizontal bar icon in the top left of the window and select 'Terminal' -> 'New Terminal'
+2. Click into the terminal window
+3. Change directory into roles/backupfile `cd roles/backup_file`
+4. Run `molecule create`. This will start a test pod for the automation to run against (defined in roles/backup_file/molecule/default/molecule.yml).
+5. Run `oc get pods` to view the test instance that was created
+6. Run `molecule verify` to run the verification against the test pod and see the failures to help guide the tasks necessary in the role.
+7. Run `molecule converge` to run the base tasks/main.yml against the pod. Notice that it only contains a 'ping' task. This demonstrates connectivity, but is not sufficient to pass the verifier.
+8. Edit roles/backup_file/tasks/main.yml to add the missing tasks
```

Comment:
I'd say it's better to have all needed tasks to have the sample complete.
To show all tasks complete successful from the verify.yaml, because now I see them as failed:
    
![screenshot-che-dogfooding apps che-dev x6e0 p1 openshiftapps com-2023 07 04-13_45_42](https://github.com/devspaces-samples/ansible-devspaces-demo/assets/1271546/3ad35766-62ef-4dfe-a1c7-be59cc7d45b1)

Reply:
Sounds good to me. Moved answers.yml content into main.yml and removed answers.yml.

Reply:
commit pushed